generate.js

```diff
@@ -101,3 +101,7 @@ fs.writeFileSync('./data.json', JSON.stringify({
   scripts: Object.keys(scripts),
   eaw: Object.keys(eaws)
 }));
+
+// Trie is serialized suboptimally as JSON so it can be loaded via require,
+// allowing unicode-properties to work in the browser
+fs.writeFileSync('./trie.json', JSON.stringify(trie.toBuffer()));
```

Comment:
Maybe encoding it as base64 would be smaller?

Reply:
Going to test it and check if that's true 😄 
Thanks!

Reply:
Yes. Good call @devongovett !

Base64 encoded trie: 11,279 bytes
Trie raw file: 8,451 bytes.

Considering they are so close, do you think should we leave them separate anyway, or use encoded base64 for both builds and get back to having only one entry point to the lib?